src/Components/Alert/Components/Form/SuggestedFilters.tsx

```diff
@@ -66,7 +66,7 @@ export const SuggestedFilters: React.FC<SuggestedFiltersProps> = ({
   return (
     <Flex justifyContent="space-between" flexDirection={"column"}>
       <Box>
-        <Text variant="sm-display">Add Filters</Text>
+        <Text variant="sm-display">Suggested Filters</Text>
```

Comment:
I also changed the title here. According to design, in should say "Suggested Filters"

Reply:
looks like Barney forgot to change the copy here- we intentionally wanted this to be Add Filters instead of suggested filters actually. No need to make this change